trigger/volume.go

```diff
@@ -0,0 +1,100 @@
+package trigger
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen/model/event"
+	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/evergreen-ci/evergreen/model/notification"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	registry.registerEventHandler(event.ResourceTypeHost, event.EventVolumeExpirationWarningSent, makeVolumeTriggers)
```

Comment:
Is there a reason this couldn't be included with the host triggers?

Reply:
host triggers assume there is a matching host (and the volume won't have a host). Adding a volume to the `hostBase` struct and conditionally fetching from a host/volume seemed messy.

Reply:
That makes sense, but then wouldn't the events need to be registered with a new event type? What do the volume events look like if they do not have a host as their resource ID?